website/management/commands/create_data_exposure_tasks.py

```
@@ -0,0 +1,209 @@
+                "target_url": "/.env",
+                "vulnerable_parameters": [],
+                "success_payload": "Find simulated SECRET_KEY and provide remediation steps",
+                "success_criteria": "User mentions rotate, revoke, secrets manager",
+                "hints": [
+                    "Search for exposed /.env",
+                    "Provide fix steps",
+                ],
+                "difficulty": "advanced",
+            },
+        },
+    ]
```

Ruff (0.14.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)